nodes/arduino_handler.py

```diff
@@ -309,7 +309,7 @@ def process_message(line):
 
 
 def connect_serial(serial_connection=None):
-    timeout_s = 1 / serial_rate_hz
+    timeout_s = 5 # 5 second serial port timeout
```

Comment:
Instead of a hacky hard coded '5', how about we make the timeout 2x the rate (actually .5 the rate, so 2 secs).